### PR TITLE
[MBL-16211][Teacher] - Extend InboxE2E with Star, Delete, Archive, unread features

### PR DESCRIPTION
**Pull request by kdeakinstructure**
Extend Teacher Inbox E2E test. - Star, Delete, Archive, unread features.

refs: MBL-16211
affects: Teacher
release note: none

### CODE REVIEW

File: apps/teacher/src/androidTest/java/com/instructure/teacher/ui/e2e/InboxE2ETest.kt [hunk 1]
```diff
@@ -108,6 +113,66 @@ class InboxE2ETest : TeacherTest() {
...
+
+        Log.d(STEP_TAG,"Filter the Inbox by selecting 'Starred' category from the spinner on Inbox Page." +
+                "Assert that the '${seedConversation[0]}' conversation is displayed because it's still starred.")
+        inboxPage.filterInbox("Starred")
+        inboxPage.assertHasConversation()
+
+        Log.d(STEP_TAG,"Click on the conversation.")
+        inboxPage.clickConversation(seedConversation[0])
+
+        Log.d(STEP_TAG, "Star the conversation and navigate back to Inbox Page.")
```
Comment (tamaskozmer) [inline apps/teacher/src/androidTest/java/com/instructure/teacher/ui/e2e/InboxE2ETest.kt:155]: Small typo. I guess this meant to be "Unstar".

**Review by tamaskozmer — APPROVED** [review]

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

File: apps/teacher/src/androidTest/java/com/instructure/teacher/ui/e2e/InboxE2ETest.kt (hunk 1 above)
Comment (kdeakinstructure) [reply apps/teacher/src/androidTest/java/com/instructure/teacher/ui/e2e/InboxE2ETest.kt:155]: Yes, thanks!

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Pull request merged**